homeassistant/components/ecobee/number.py

```diff
@@ -0,0 +1,82 @@
+"""Support for using number with ecobee thermostats."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import EcobeeBaseEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the ecobee thermostat number entity."""
+    data = hass.data[DOMAIN]
+    entities = []
+    _LOGGER.debug("Adding min time ventilators numbers (if present)")
+    for index, thermostat in enumerate(data.ecobee.thermostats):
+        if thermostat["settings"]["ventilatorType"] != "none":
+            _LOGGER.debug("Adding %s's ventilator min times number", thermostat["name"])
+            entities.append(
+                EcobeeVentilatorMinTime(
+                    data,
+                    index,
+                    "home",
+                    "ventilatorMinOnTimeHome",
+                    data.ecobee.set_ventilator_min_on_time_home,
+                )
+            )
+
+            entities.append(
+                EcobeeVentilatorMinTime(
+                    data,
+                    index,
+                    "away",
+                    "ventilatorMinOnTimeAway",
+                    data.ecobee.set_ventilator_min_on_time_away,
+                )
+            )
```

Comment:
Since I have a feeling there is going to be more of these coming we should use `NumberEntityDescription` and create a `set_fn`

Example in homeassistant/components/deconz/number.py

Reply:
There could only be 2 type of `EcobeeVentilatorMinTime` Home and Away. (This is the way the integration works) 

If we add other type of numbers, I can't garante that they will require the same field types. 

Do you still think it is worth doing a `EcobeeVentilatorMinTime`

Reply:
> If we add other type of numbers, I can't garante that they will require the same field types.
> 
> Do you still think it is worth doing a `EcobeeVentilatorMinTime`

Yes

Reply:
Thanks